scheduler/scheduler/config/configs.go

```diff
@@ -1,74 +1,84 @@
 package config
 
+import (
+	"time"
+
+	"github.com/twitter/scoot/scheduler/server"
+	"github.com/twitter/scoot/workerapi/client"
+)
+
 // SchedulerConfigs the map of available configurations
-var SchedulerConfigs = map[string]string{
+var ServiceConfigs = map[string]ServiceConfig{
 	"default":      defaultConfig,
 	"local.local":  localLocal,
 	"local.memory": localMemory,
 }
 
-// defaultConfig the configuration values that are used for nil parts of a specific configuration
-const defaultConfig = `
-{
-	"Cluster": {
-	  "Type": "memory"
+// defaultConfig the configuration values that are used for nil parts of a specific configuration and for integration tests
+var defaultConfig = ServiceConfig{
+	ClusterConfig{
+		Type: "memory",
 	},
-	"Workers": {
-	  "Type": "local"
+	client.WorkersClientConfig{
+		Type: "local",
 	},
-	"SchedulerConfig": {
-	  "Type": "stateful",
-	  "MaxRetriesPerTask" : 0,
-	  "DefaultTaskTimeout" : "30m"
+	server.SchedulerConfig{
+		MaxRetriesPerTask:   0,
+		DefaultTaskTimeout:  time.Duration(30) * time.Minute,
+		RunnerRetryTimeout:  DefaultRunnerRetryTimeout,
+		RunnerRetryInterval: DefaultRunnerRetryInterval,
+		ReadyFnBackoff:      DefaultReadyFnBackoff,
+	},
+	SagaLogConfig{
+		Type: "memory",
 	},
-	"SagaLog": {
-	  "Type": "memory"
-	}
 }
-`
 
 // localLocal config for local.local - !!! make sure this constant is added to SchedulerConfigs map above !!!
```

Comment:
we could avoid this situation now by defining the configs inline directly in the map now